### PR TITLE
Setup bikeshed

### PR DESCRIPTION
**Pull request by chcunningham**
Skeleton for bikeshed spec and Makefile with some early idl.

### CODE REVIEW

**Comment by chcunningham** [comment]
Hey Peter. Just getting the ball rolling. No actual spec language written at this point. I grabbed the VideoDecoder interface you checked in. Considered grabbing the Encoder parts as well, but that seems less complete (VideoEncoder isn't yet defined). 

Next thing to do is to setup gh-pages deployment. Can you grant me admin on the repo? I'll need to generate the deployment key.

**Comment by chcunningham** [comment]
Also, had to add an Editor: for bikeshed to compile. @padenot would you like to be listed here as well?

**Comment by padenot** [comment]
Yep, thanks.

File: index.bs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    [Exposed=(Window)]
+    dictionary VideoDecoderInitParameters {
+      required DOMString codec;
+      required DOMString profile;
+
+      // These are the "coded size", not the "visible size"
+      required unsigned long width;
+      required unsigned long height;
+    };
+  </pre>
```
Comment (pthatcherg) [inline index.bs]: For VideoEncoder, I think you can add a subset I've got in my CL for now:

interface VideoEncoder {
    Promise<void> Initialize(VideoEncoderInitParameters params);
    void Close();

    readonly attribute WritableStream writable;  // of VideoFrame
    readonly attribute ReadableStream readable;  // of EncodedVideoFrame
};

enum VideoEncoderHardwareMode {
  "hardware",
  "software"
};

dictionary VideoEncoderInitParameters {
  VideoEncoderHardwareMode hardware_mode;
  DOMString codec;
  DOMString profile;

  unsigned long visible_width;
  unsigned long visible_height;

  unsigned long bitsPerSecond;
  double framesPerSecond;

  double maxFramesBetweenKeyFrames;
};

Comment (chcunningham) [reply index.bs]: Done!

**Pull request merged**